src/shared/SliderToggle.tsx

```diff
@@ -0,0 +1,146 @@
+import * as React from 'https://esm.sh/react@19.0.0?target=es2022';
+
+const { useEffect, useState } = React;
+
+export enum FilterLevel {
+  Off = 'off',
+  Default = 'default', 
+  Max = 'max'
+}
```

Comment:
Don't use TypeScript enum. Use union instead. See https://www.kabuku.co.jp/developers/good-bye-typescript-enum for detail